admin/src/views/Agents.vue

```diff
@@ -9,55 +9,94 @@
+          <el-radio-group v-model="agentCreationOption"
+            @change="isWithImportAgentCreation = !isWithImportAgentCreation">
+              <el-tooltip
+                class="item"
+                effect="dark"
+                content="Manual Agent Creation"
+                placement="top-start"
+              >
+                <el-radio label="manualCreation">
+                  Manual Agent Creation
+                </el-radio>
+              </el-tooltip>
+              <el-tooltip
+                class="item"
+                effect="dark"
+                content="Importing file for Agent creation"
+                placement="top-start"
+              >
+                <el-radio label="creationWithImport">
+                  Import File
+                </el-radio>
+              </el-tooltip>
+        </el-radio-group>
         </el-form-item>
+        <div v-if="!isWithImportAgentCreation">
+          <el-form-item label="NLU Conf">
+            <el-radio-group v-model="newAgent.configType" @change="selectConf">
+              <el-tooltip
+                class="item"
+                effect="dark"
+                content="Recommended when you have more than 1000 training data"
+                placement="top-start"
+              >
+                <el-radio label="supervised_embeddings">
+                  Supervised Embeddings
+                </el-radio>
+              </el-tooltip>
+              <el-tooltip
+                class="item"
+                effect="dark"
+                content="Recommended when you need a specific nlu pipeline"
+                placement="top-start"
+              >
+                <el-radio label="specific_pipeline">
+                  Specific Pipeline
+                </el-radio>
+              </el-tooltip>
+            </el-radio-group>
+          </el-form-item>
+          <el-form-item label="Language" :class="[isHideSpecificPipeline ? '' : 'displayNone']">
+            <el-select v-model="newAgent.language" placeholder="please select your zone">
+              <el-option label="English" value="en"></el-option>
+              <el-option label="French" value="fr"></el-option>
+            </el-select>
+          </el-form-item>
+          <el-form-item label="Specific pipeline" :class="[isHideSpecificPipeline ? 'displayNone' : '']">
+            <el-input type="textarea" :rows="10" v-model="newAgent.pipeline"></el-input>
+          </el-form-item>
+          <el-form-item label="Training Data">
+            <el-input type="textarea" :rows="7" v-model="newAgent.trainingData"></el-input>
+          </el-form-item>
+          <el-form-item label="Responses">
+            <el-input type="textarea" :rows="7" v-model="newAgent.responses"></el-input>
+          </el-form-item>
+        </div>
+        <div v-if="isWithImportAgentCreation">
+          <el-form-item>
+            <el-upload
+            class="upload-demo"
+            ref="upload"
```

Comment:
Unused ref, to delete

Reply:
is that resolved ?

Reply:
Ref attribute is used to enable or disable button